src/sign.ts

```diff
@@ -16,6 +21,16 @@ function get_sign_options(options: any[]): void {
   }
 
   options.push(`--name=${ARTIFACT_NAME}`);
+
+  if (
+    !ENABLE_GITHUB_ACTION_TOKEN &&
+    !(CERTIFICATE_OIDC_ISSUER && SOURCE_REPOSITORY_REF)
+  ) {
```

Comment:
Should the options be added below when the inputs are present and ENABLE_GITHUB_ACTION_TOKEN is false?

```ts
options.push(`--source-repository-ref=${SOURCE_REPOSITORY_REF}`);
options.push(`--certificate-oidc-issuer=${CERTIFICATE_OIDC_ISSUER}`);
```